### DISCUSSION TITLE
Issue with toolbar_control

### DISCUSSION THREAD

**Discussion by felwind14** [giswqs/geog-312#115] [Q&A] [answered]
Dear @giswqs 

I am trying to display the toolbar_control on my vs_code but  it is not working when I try in jupiterlab it does, however I see in the video for you the toolbar_control it is getting displayed on your vs code any hint how to solve this?

Thank you

**Comment by giswqs** [comment]
What do you mean by "not working"? Does the button show up in VS Code?

**Reply by felwind14** [reply]
Yes this is. The icon does not appear maybe it is wsl

**Comment by giswqs** [comment, marked as answer]
Try creating a new conda env to install leafmap.
